README.adoc

```diff
@@ -36,38 +36,39 @@ Learn how to create, use and cache HTTP session data for your application.
 
 === What is a session?
 // *Session* +
-On the internet, a web server doesn't know who you are, or what you do, 
-because it's just processing stateless HTTP requests. A session is a way to store 
+On the internet, a web server doesn't know who you are or what you do
+because it's processing stateless HTTP requests. An HTTP session provides a way to store
 information to be used across multiple requests.
-Session variables store user information like username, or items in a shopping cart.
- By default, session variables will timeout after 30 minutes of being unused.
-Cookies, which also store user
-information, are maintained on a client's computer, whereas session variables are maintained on a web server. For security reasons, using
-session is preferred over using cookies for sensitive data. Session hides data
-from users. Cookies can be manipulated by a
-savvy user to make fake requests to your site.
-
-=== What is session persistence? 
+Session variables store user information like user name or items in a shopping cart.
+By default, session variables will timeout after 30 minutes of being unused.
+Cookies, which also store user information, are maintained on a client's computer,
+whereas session variables are maintained on a web server. For security reasons,
+an HTTP session is preferred over cookies when use with sensitive data.
```

Comment:
Change
"For security reasons, an HTTP session is preferred over cookies when use with sensitive data."
To
"For security reasons, an HTTP session is preferred over cookies when used with sensitive data."

Reply:
addressed by https://github.com/OpenLiberty/draft-guide-sessions/commit/c2d7fd4e2cdd07105289d9b17fbb4ab1457628f2